pyanaconda/packaging/yumpayload.py

```diff
@@ -566,8 +566,8 @@ def updateBaseRepo(self, fallback=True, root=None, checkmount=True):
                                  proxyurl=proxyurl, sslverify=sslverify)
                 self._addAddons(self._yum.repos.getRepo(BASE_REPO_NAME), url, proxyurl, sslverify)
             except (MetadataError, PayloadError) as e:
-                log.error("base repo (%s/%s) not valid -- removing it",
-                          method.method, url)
+                log.error("base repo (%s/%s) not valid -- removing it\nreason: %s",
+                          method.method, url, e)
```

Comment:
The \n in there is going to mess up how the logs look. either log it as a separate line, or "-- removing it because: %s"